src/components/Page/components/Header/types.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  AppBridgeAction,
+  DestructableAction,
+  DisableableAction,
+  LoadableAction,
+} from '../../../../types';
+
+export interface HeaderPrimaryAction
```

Comment:
This type is used both within the Page and Page.Header. In order to avoid potential for circular dependencies due to components referencing each other things like utility function and types should live at the highest level that need them.

Can you pull this up and put it in src/components/Page/types.ts instead?

Reply:
I'll move it up to `Page/types.ts`